src/actions/layerEdit.js

```diff
@@ -179,10 +179,10 @@ export const setPeriodName = (periodName) => ({
 })
 
 // Set period type (thematic)
-export const setPeriodType = (periodType, clearPeriod = true) => ({
+export const setPeriodType = (periodType, keepPeriod) => ({
```

Comment:
This function is used in TrackedEntities PeriodTypeSelect, but without the second argument. Will this change the behavior for the TE Dialog since the default will be reversed?

Reply:
It should not be affected: if we the second argument is not passed we will clear the period, and this is happening in `src/reducers/layerEdit.js`.